ts/util/attachmentsUtil.ts

```diff
@@ -240,10 +255,10 @@ export async function autoScale<T extends { contentType: string; blob: Blob }>(
   do {
     i -= 1;
     if (DEBUG_ATTACHMENTS_SCALE) {
-      // window.log.info(`autoscale iteration: [${i}] for:`, attachment);
+      window.log.info(`autoscale iteration: [${i}] for:`, JSON.stringify(readAndResizedBlob.size));
```

Comment:
Should this not be `window.log.debug` ?

Reply:
updated all of them but I still want it not enabled whenever we turn the debug flag on so I kept the boolean too